Havit.Blazor.Components.Web.Bootstrap/TreeViews/HxTreeViewItem.razor.css

```diff
@@ -0,0 +1,44 @@
+.hx-tree-item {
+  display: flex;
+  cursor: pointer;
+  align-items: center;
+  padding: 5px;
+  font-size: 14px;
+  border-radius: var(--hx-treeview-item-border-radius);
+  background-color: var(--hx-treeview-item-background);
+  color: var(--hx-treeview-item-color);
+}
+.hx-tree-item.selected {
+  background-color: var(--hx-treeview-item-selected-background);
+  color: var(--hx-treeview-item-selected-color);
+  transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out;  
+}
+.hx-tree-item:hover  {
+  background-color: var(--hx-treeview-item-hover-background);
+  color: var(--hx-treeview-item-hover-color);
+  transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out;
+} 
+.hx-tree-item .hx-tree-title-value {
```

Comment:
@crdo, @sshulik, should we use `.hx-tree-title-value` or is `.hx-tree-item-title` a better name?

Reply:
consolidated